plugins/kubernetes/test/models/kubernetes/resource_test.rb

```diff
@@ -165,7 +165,7 @@ def assert_create_and_delete_requests(**args, &block)
           end
         end
 
-        it "explains why it is a bad idea" do
+        it "allows updating when opting out" do
           old = {spec: {selector: {matchLabels: {foo: "baz"}}}}
           resource.template[:metadata][:annotations] = {"samson/allow_updating_match_labels": "true"}
```

Comment:
Please document this annotation in readme.

Reply:
already documented